packages/core/src/connectors/github/index.ts

```diff
@@ -57,3 +57,29 @@ export const getAccessToken: GetAccessToken = async (code) => {
     }>();
   return accessToken;
 };
+
+export const getUserInfo: GetUserInfo = async (accessToken: string) => {
+  const {
+    id,
+    avatar_url: avatar,
+    email,
+    name,
+  } = await got
+    .get(userInfoEndpoint, {
+      headers: {
+        authorization: `token ${accessToken}`,
```

Comment:
Bearer?

Reply:
It is `token`: https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps#3-use-the-access-token-to-access-the-api